e2e_playwright/st_map.py

```diff
@@ -70,3 +70,10 @@
     size="size",
     use_container_width=False,
 )
+
+
+"""
+### Simple map with defined width and height
+"""
+
+st.map(simple_map_df, width=200, height=250, use_container_width=False)
```

Comment:
for my own understanding: is it possible to just pass one of them? If yes, is that then set accordingly and the other one is auto-determined?

Reply:
Yes, you can pass just one and the other will be auto-determined much like it does today!